src/uikit.cs

```diff
@@ -2360,6 +2360,7 @@ interface UIViewPropertyAnimator : UIViewImplicitlyAnimating, NSCopying
 	
 		[Static]
 		[Export ("runningPropertyAnimatorWithDuration:delay:options:animations:completion:")]
+		[Async]
```

Comment:
I do not think we can Async'ify this one according to [docs](https://developer.xamarin.com/guides/cross-platform/macios/binding/binding-types-reference/#AsyncAttribute):

> You can use the `[Async]` attribute on methods that **return void** and whose **last** argument is a callback.

Also not entirely sold on this Async API, [docs](https://developer.apple.com/reference/uikit/uiviewpropertyanimator/1648367-runningpropertyanimatorwithdurat) says the following:

> Creates and returns an animator object that begins running its animations immediately.

This means it will return immediately but the callback is not guaranteed to be done at the time this method returns 

> The block to execute when the animations finish. You can use this block to perform any final actions. This block has no return value and takes the following parameter:
>
> finalPosition:
> The ending position of the animations. Use this value to determine whether the animations stopped at the beginning, end, or somewhere in the middle.